src/PRo3D.Viewer/TraverseApp.fs

```diff
@@ -347,34 +338,64 @@ module TraverseApp =
 
         sols
 
+    // Function to split the string into chunks of numbers and non-numbers
+    let splitString (input: string) =
+        Regex.Matches(input, @"\D+|\d+")
+        |> Seq.cast<Match>
+        |> Seq.map (fun m -> m.Value)
+        |> Seq.toList
+    
+    // Function to compare two strings with natural sorting
+    let compareNatural (left: AdaptiveTraverse) (right: AdaptiveTraverse) =
+        let s1 = left.tName|> AVal.force
+        let s2 = right.tName|> AVal.force
+
+        let rec compareParts (parts1: string list) (parts2: string list) =
+            match parts1, parts2 with
+            | [], [] -> 0
+            | [], _ -> -1
+            | _, [] -> 1
+            | h1::t1, h2::t2 ->
+                let isNum1 = Int32.TryParse(h1)
```

Comment:
bit nitpicking but thinking about this one...

would creating a list of char and using default comparison work also?
```
> [1;2;3] < [2;3;4];;
val it: unit = ()

> val it: bool = true

```

also, how about having the tool general purpose, e.g. of string and provide it with `(adaptiveTraverse1.Current, adaptiveTraverse2.Current) ||> AVal.map2 (fun l r -> naturalSort l.tName r.tName)`?